docs/tutorials/projects/guardian.md

```diff
@@ -255,7 +255,7 @@ scp labels.txt pi@guardian.local:/home/pi/labels.txt
 Next, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
 Click on the **Services** subtab and navigate to the **Create service** menu.
 
-1. **Add a ML service.**
+1. **Add the ML Service.**
```

Comment:
So you can now add multiple ML services. We decided that we'll capitalize Vision Service and ML Model Service (which that's actually a problem here I didn't use the full name) when we talk about the service as a concept but when we talk about an instance we do not capitalize. Because with you suddenly being able to have multiple it felt strange to capitalize it so much. Thank you for checking though!

```suggestion
1. **Add a ML model service.**
```